frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -135,6 +121,27 @@
                   onRunSubmitted(code, selectedLanguage)
               "
             ></omegaup-arena-runsubmit-popup>
+            <omegaup-quality-nomination-promotion
+              :can-nominate-problem="nominationStatus.canNominateProblem"
+              :dismissed="nominationStatus.dismissed"
+              :dismissed-before-a-c="nominationStatus.dismissedBeforeAC"
```

Comment:
```suggestion
              :dismissed-before-ac="nominationStatus.dismissedBeforeAc"
```

creo que es mejor que se llame `Ac` para evitar el `a-c`.

Reply:
Ahorita sólo voy a arreglar el nombre en este lugar para no hacer más grande el PR. En otro cambio lo arreglo de raíz